xmipptomo/protocols/protocol_cltomo.py

```diff
@@ -23,14 +23,13 @@
 # *  e-mail address 'scipion@cnb.csic.es'
 # *
 # **************************************************************************
+import os
```

Comment:
Not sure if this protocol is properly working. This protocol was in xmipp3 plugin as a first try for subtomogram classification. When plugin xmipptomo was created, this protocol was migrated here, but I think for the moment is a 'work in progess in pause', maybe we can leave it just in devel and not release it @pconesa? (same with its viewer)